dumpSTR/README.md

```diff
@@ -50,6 +50,16 @@ GangSTR-specific call-level filters:
 * **`--filter-spanbound-only`**: Filter loci where only spanning or bounding reads were identified
 * **`--filter-badCI`**: Filter regions where the ML estimate is not in the CI
 
+## Basic dumpSTR command 
+
+```
+dumpSTR \
+        --vcf /storage/mikhail/100919_BXD_all_drive1_drive3_rep_stutt_model/result/4512-JFI-0366_BXD196/res_chr1.vcf.gz \
```

Comment:
change to have relative path name within the repository
Goal is that someone can git clone the repo, copy the command, and it will just work